python/theme_Motoman.py

```diff
@@ -46,18 +46,19 @@
 repo_match(repository, match, name_control, "control")
 
 
-match = r'\WP\d+' #regex for position use ex P00095
+match = r'\bP\d{5}' #regex for position use ex P00095
```

Comment:
The header of this file cites ABB and not Yaskawa, if you can fix it in this PR.